data/serializer/src/test/resources/tech/pegasys/teku/api/response/v1/schema/BlockEvent.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type" : "object",
+  "id" : "urn:jsonschema:tech:pegasys:teku:api:response:v1:BlockEvent",
+  "properties" : {
+    "block" : {
+      "type" : "object",
+      "id" : "urn:jsonschema:org:apache:tuweni:bytes:Bytes32",
+      "properties" : {
+        "empty" : {
+          "type" : "boolean"
+        },
+        "zero" : {
+          "type" : "boolean"
+        }
+      },
+      "required" : true
+    },
```

Comment:
This looks like it's looking into the Bytes32 type.  It doesn't matter because we don't actually output this content anywhere but it does mean these definitions will change quite a lot when we switch to type definitions.